config/packages/api_platform.yaml

```diff
@@ -32,9 +32,6 @@ api_platform:
     patch_formats:
         json: ['application/merge-patch+json']
 
-    swagger:
-        versions: [3]
```

Comment:
Is it defaulting to version 3?

Reply:
"By default, OpenAPI v3 is used" - https://api-platform.com/docs/core/openapi/